sysid-application/src/integrationtest-utils/native/cpp/IntegrationUtils.cpp

```diff
@@ -58,7 +62,11 @@ void Connect(NT_Inst nt, NT_Entry kill) {
   }
 }
 
-void KillNT(NT_Inst nt, NT_Entry kill) {
+void KillNT(NT_Inst nt, NT_Entry kill, std::string* logContent) {
+  // Before killing sim, store any captured console output.
+  auto& capturedOutput = *logContent;
+  capturedOutput = ::testing::internal::GetCapturedStdout();
```

Comment:
You could store the std::string, then return it by value instead. That means the caller gets their own copy, RVO kicks in, and you won't need the static global string.